SQL/New_patches/2018-09-07_EEG_tables.sql

```diff
@@ -0,0 +1,259 @@
+  ('derivatives', 'derivative/processed dataset');
+
+-- Insert into physiological_channel_type
+INSERT INTO physiological_channel_type
+  (`ChannelType`, `ChannelDescription`)
+  VALUES
+  ('EEG',              'ElectoEncephaloGram: EEG sensors'                    ),
+  ('VEOG',             'Vertical ElectroOculoGram (eyes)'                    ),
+  ('HEOG',             'Horizontal ElectOculoGram'                           ),
+  ('EOG',              'Generic EOG channel, if HEOG or VEOG information not available'),
+  ('ECG',              'ElectroCardioGram (heart)'                           ),
+  ('EMG',              'ElectroMyoGram (muscle)'                             ),
+  ('TRIG',             'System Triggers'                                     ),
+  ('REF',              'Reference electrode'                                 ),
+  ('GRD',              'Ground electrode'                                    ),
+  ('MISC',             'Miscellaneous'                                       ),
+  ('MEGMAG',           'MEG magnetometer'                                    ),
+  ('MEGGRADAXIAL',     'MEG axial gradiometer'                               ),
+  ('MEGGRADPLANAR',    'MEG planar gradiometer'                              ),
+  ('MEGGREFMAG',       'MEG reference magnetometer'                          ),
+  ('MEGREFGRADAXIAL',  'MEG reference axial gradiometer'                     ),
+  ('MEGREFGRADPLANAR', 'MEG reference planar gradiometer'                    ),
+  ('MEGOTHER',         'Any other type of MEG sensor'                        ),
+  ('ECOG',             'Electrode channel: electrocortigogram (intracranial)'),
+  ('SEEG',             'Electrode channel: stereo-electroencephalogram (intracranial)'),
+  ('DBS',              'Electrode channel: deep brain stimulation (intracranial)'),
+  ('AUDIO',            'Audio signal'                                        ),
+  ('PD',               'Photodiode'                                          ),
+  ('EYEGAZE',          'Eye Tracker gaze'                                    ),
+  ('PUPIL',            'Eye Tracker pupil diameter'                          ),
+  ('SYSCLOCK',         'System time showing elapsed time since trial started'),
+  ('ADC',              'Analog to Digital input'                             ),
+  ('DAC',              'Digital to Analog input'                             ),
+  ('HLU',              'Measure position of head and head coils'             ),
+  ('FITERR',           'Fit error signal from each head localization coil'   ),
+  ('OTHER',            'Any other type of channel'                           );
+
+
+-- Insert into physiological_modality
+INSERT INTO physiological_modality
+  (PhysiologicalModality)
+  VALUES
+  ('eeg'),
+  ('meg'),
+  ('ieeg');
+
+
+-- Insert into physiological_status_type
+INSERT INTO physiological_status_type
+  (ChannelStatus)
+  VALUES
+  ('good'),
+  ('bad');
+
+-- Insert into ImagingFileTypes
+INSERT INTO ImagingFileTypes
```

Comment:
Is this table shared with the imaging browser? If not the name is misleading (also, the capitalization style is different than all the other tables in this PR.)

Reply:
Indeed, this is the same table than the one used for the MRI side. It did not make any sense to me to create an additional file type table just for EEG. The capitalization is due to the fact that this is what it is currently called. Renaming and normalization of that table should probably be independent of this PR.